src/sentry/models/projectownership.py

```diff
@@ -376,7 +376,7 @@ def process_resource_change(instance, change, **kwargs):
     autoassignment_types = ProjectOwnership._get_autoassignment_types(instance)
     if len(autoassignment_types) > 0:
         GroupOwner.invalidate_autoassigned_owner_cache(instance.project_id, autoassignment_types)
-
+    GroupOwner.invalidate_assignee_exists_cache(instance.project.id)
```

Comment:
This is the only change, and it this runs as a signal when ProjectOwnership updates/deletes.

Reply:
I will look into adding a test though :+1:

Reply:
got it, makes sense! thanks.